includes/Classifai/Providers/Azure/ComputerVision.php

```diff
@@ -63,6 +67,37 @@ private function get_default_settings() {
 		];
 	}
 
+	/**
+	 * Returns an array of fields enabled to be set to store image captions.
+	 * Returns `false` if no fields are selected.
+	 *
+	 * @return array|boolean
+	 */
+	public function get_alt_text_settings() {
+		$settings       = $this->get_settings();
+		$enabled_fields = array();
+
+		if ( ! isset( $settings['enable_image_captions'] ) ) {
+			return false;
```

Comment:
```suggestion
			return array();
```

I noticed that when no fields are set, this method sometimes returns `false` and sometimes an empty array. It would be good to be consistent.

This might allow you to simplify `generate_alt_tags()` a little as the method could assume an array and lose the repeated `is_array()` check.

To reproduce:
* delete the option from the database
* var_dump shows false
* save the image processing options page without enabling the descriptive text for any option
* var_dump of this method shows `array()`